feature-group-definitions/search-input-type.dist.yml

```diff
@@ -0,0 +1,20 @@
+# Generated from: search-input-type.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: '<input type="search">'
+spec: https://html.spec.whatwg.org/multipage/input.html#text-(type=text)-state-and-search-state-(type=search)
+caniuse: input-search
+status:
+  baseline: high
+  baseline_low_date: 2015-07-29
+  baseline_high_date: 2018-01-29
+  support:
+    chrome: "5"
```

Comment:
https://caniuse.com/input-search has slightly different data. I'll back this out.